shared/wallets/confirm-form/participants/container.js

```diff
@@ -8,21 +8,20 @@ const mapStateToProps = state => {
   const build = state.wallets.building
   const built = state.wallets.builtPayment
 
-  let recipientUsername = built.toUsername
+  const recipientType = build.recipientType
+  const recipientUsername = (recipientType === 'keybaseUser' && build.to) || ''
   const userInfo = state.users.infoMap.get(recipientUsername)
-  const recipientFullName = userInfo ? userInfo.fullname : ''
+  let recipientFullName = userInfo ? userInfo.fullname : ''
   const fromAccount = getAccount(state, built.from)
 
-  const recipientType = build.recipientType
   let recipientStellarAddress
   let recipientAccountIsDefault
   let recipientAccountName
   let recipientAccountAssets
   if (recipientType === 'keybaseUser') {
```

Comment:
I think you can remove this block entirely

Reply:
which block?

Reply:
Can replace
```
if (recipientType === 'keybaseUser') {
    if (build.to.includes('@')) {
      // this is an sbs assertion
      recipientFullName = ''
    }
} else
```
with
``` 
if (recipientType !== 'keybaseUser')
```
I don't think we'll ever find a fullName in `userInfo` for sbs, and if we do I don't think it'd be inaccurate to show.

Reply:
ah nice thanks